### DISCUSSION TITLE
Uninstall Open Shell before Win 11 upgrade?

### DISCUSSION THREAD

**Discussion by yorktraf** [Open-Shell/Open-Shell-Menu#1204] [General]
We recently went through a lengthy migration/upgrade to new PC en route to enabling Win11 free upgrade from Win10 (first old PC's C drive was too small and the PC into which new C drive was installed itself proved problematic/required replacement.) 

We've used first Classic Shell and lately, Open Shell throughout our time on Win10. I'm wondering if my Win 11 upgrade with preservation of applications and files will go better if I FIRST uninstall Open Shell before the upgrade (and then reinstall once we're safely on Win 11?)

I'll gladly put up with a short period on an unfamiliar interface if the Win11 install is likely to go more smoothly. We had no computer for 3 weeks while all this system imaging and migration rolled out and I just want the least possible down time and technical problems in this next phase, especially as I'll be doing it myself.

**Comment by blackcrack** [comment]
you have a good running Win10, OpenShell works pretty well in Win10, so, my personally hint.., stay on Win10.
hmm.. if your Computer more older, do you have maybe not TPM 2.0 for Win11 or... Win11 is not Compatible for your Computer, this is also an Reason what you have remember.. so, my personally Opinion, let the Fingers away from Win11. I work with my Win7 so long up to it decay.. Libreoffice works, Gimp works the most Windowsprogramms works, Win7 is x86_64 bit and i have store the most Programs from the Win7 Times.. and the most actual programs supports still Win7 so, what's that nonsense, we have to take ever the newest WinNT or we are not supported.. this is mental blackmail and nothing else .. and the best, if you use some hints or extra patches, works the commercial patches and updates too, becuse up to 2023 is the Win7 system and security updates still avail and is updated all single month up to 2023 ! so, What have you to update.. later exist an Antivir and Softwarefilrewall also.. 
best

**Comment by ge0rdi** [comment]
Hello @yorktraf .

There should be no need to uninstall `Open-Shell` before Windows upgrade.
It shouldn't interfere with the upgrade process.

Though note that `Open-Shell` is not officially compatible with Windows 11.
Many things don't work there out of the box.

You can go through discussions and issues related to Windows 11 to see the picture.

**Comment by tallman69** [comment]
Until Open-Shell is officially compatible with Windows 11, I would recommend installing ExplorerPatcher which gives Open-Shell more of a Windows 10 environment to operate in.  I have had virtually no trouble with Open-Shell on Windows 11 when also running ExplorerPatcher.  Hope all goes well.
